featuretools/tests/primitive_tests/test_features_deserializer.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+
+from ..testing_utils import make_ecommerce_entityset
+
+import featuretools as ft
+from featuretools.feature_base.features_deserializer import (
+    FeaturesDeserializer
+)
+
+SCHEMA_VERSION = "1.0.0"
```

Comment:
It can be. I duplicated it here to "test" that the correct value is set in `features_serializer.py`. But I'll remove it if you don't think testing the value of the constant is necessary.

Reply:
So it's testing that featuretools has `SCHEMA_VERSION` is set to the value we expect?

Reply:
Yes.

Reply:
Let's test that featuretools version is what we expect as well.  But I think we should only hardcode `SCHEMA_VERSION` in one location instead of two, so that it's easier to update.